compiler/qsc_eval/src/val.rs

```diff
@@ -263,6 +263,14 @@ impl Value {
         v
     }
 
+    #[must_use]
+    pub fn get_double(&self) -> f64 {
```

Comment:
Suggest renaming to `unwrap_double` to stick with the naming convention here.

Reply:
We discussed this with Stefan. There's already unwrap_double, but it consumes the argument (so requires copy/move of an argument in my case). In my case borrowing (instead of consuming) works just fine and that's why I need this get function.